components/PralineBox/box.styled.js

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+
+export const StyledWrapper = styled.div`
+  display: grid;
+  background-color: grey;
+  grid-template-columns: 30% 30% 30%;
+  margin: 20px;
+  border-radius: 16px;
+  gap: 10px;
+  print-color-adjust: exact;
+  -webkit-print-color-adjust: exact;
```

Comment:
You don't need to add the prefix "webkit" in your css. It's added automatically in the build step.